diff -Nrub a/drivers/misc/tracker_buttons.c b/drivers/misc/tracker_buttons.c
--- a/drivers/misc/tracker_buttons.c	2014-01-05 10:05:56.000000000 -0500
+++ b/drivers/misc/tracker_buttons.c	2014-01-09 10:27:12.000000000 -0500
@@ -44,7 +44,7 @@
 };
 
 int debounce_count = 3;
-int gpio_value[4]={1,1,1,1};
+int gpio_value[4]={0,0,0,0};
 int gpio_stable[4]={0,0,0,0};
 int last_press_time;
 
@@ -123,7 +123,7 @@
 	unsigned char buff;
 	for (i=0;i<4;i++)
 	{
-		value = gpio_get_value(gpioLD[i]);
+		value = !gpio_get_value(gpioLD[i]);
 		if (value != gpio_value[i])
 		{
 			gpio_stable[i]++;
